### PR TITLE
Wise Batch and OTT

### PR DESCRIPTION
**Pull request by kewitz**
Resolve https://github.com/opencollective/opencollective/issues/4292
Resolve https://github.com/opencollective/opencollective/issues/4307

### CODE REVIEW

File: server/paymentProviders/transferwise/index.ts [hunk 1]
```diff
@@ -138,6 +208,99 @@ async function payExpense(
...
+  expenses: Array<typeof models.Expense>,
+): Promise<BatchGroup> {
+  expenses.forEach(e => {
+    if (!e.PayoutMethod) {
+      throw new BadRequest('payExpenseBatch: Expense is missing PayoutMethod', undefined, { expenseId: e.id });
+    }
+  });
+
+  const [connectedAccount] = await host.getConnectedAccounts({
+    where: { service: 'transferwise', deletedAt: null },
```
Comment (znarf) [inline server/paymentProviders/transferwise/index.ts:228]: Do we really need `{ deletedAt: null }`? Isn't this the default in Sequelize?

File: server/graphql/v2/enum/ExpenseProcessAction.ts [hunk 2]
```diff
@@ -19,6 +19,9 @@ export const ExpenseProcessAction = new GraphQLEnumType({
     SCHEDULE_FOR_PAYMENT: {
       description: 'To schedule the expense for payment',
     },
+    UNSCHEDULE_PAYMENT: {
+      description: 'To uschedule the expense payment',
+    },
```
Comment (znarf) [inline server/graphql/v2/enum/ExpenseProcessAction.ts:24]: Why not `UNSCHEDULE_FOR_PAYMENT`?

File: server/graphql/v2/enum/ExpenseProcessAction.ts [hunk 3]
```diff
@@ -19,6 +19,9 @@ export const ExpenseProcessAction = new GraphQLEnumType({
     SCHEDULE_FOR_PAYMENT: {
       description: 'To schedule the expense for payment',
     },
+    UNSCHEDULE_PAYMENT: {
+      description: 'To uschedule the expense payment',
```
Comment (znarf) [inline server/graphql/v2/enum/ExpenseProcessAction.ts:23]:
```suggestion
      description: 'To unschedule the expense payment',
```

File: server/graphql/v2/enum/ExpenseProcessAction.ts (hunk 2 above)
Comment (kewitz) [reply server/graphql/v2/enum/ExpenseProcessAction.ts:24]: I'm not sure if you can _unschedule for_ something, doesn't sound right. @alanna can you confirm?

File: server/paymentProviders/transferwise/index.ts (hunk 1 above)
Comment (kewitz) [reply server/paymentProviders/transferwise/index.ts:228]: True. 🎯

**Pull request merged**